ansible/roles/ocp4-workload-ccnrd/tasks/install-mta.yaml

```diff
@@ -1,31 +1,31 @@
 ---
-# deploy rhamt
-- name: Deploy RHAMT
+# deploy mta
+- name: Deploy MTA
   shell: oc process -f - | oc create -n labs-infra  -f -
   args:
-    stdin: "{{ lookup('file', './files/rhamt-template.yaml') }}"
+    stdin: "{{ lookup('file', './files/mta-template.yaml') }}"
 
-- name: TODO - scale RHAMT executor to quarter the number of users
+- name: scale MTA executor to quarter the number of users
   shell: |
-         oc scale dc/rhamt-web-console-executor --replicas={{ ((num_users|int) / 3) | int }} -n labs-infra
+         oc scale dc/mta-web-console-executor --replicas={{ ((num_users|int) / 3) | int }} -n labs-infra
 
-- name: remove liveness and readiness for RHAMT web console
+- name: remove liveness and readiness for MTA web console
   shell: |
-         oc set probe dc/rhamt-web-console  -n labs-infra --remove --readiness --liveness
+         oc set probe dc/mta-web-console  -n labs-infra --remove --readiness --liveness
```

Comment:
Please convert this to use `k8s` module. You can probably merge this with the previous task to do it in one with `strategic-merge` or `apply` in module.

Reply:
Any example to convert oc set probe using k8 module?

Reply:
actually before I go down that path...why do you not want any liveness or readiness probes? That seems like not a good idea to remove them unless they are causing some kind of issue?

Reply:
because the pod never comes up for the liveness and readiness check.

Reply:
@stencell I'm trying to test without oc set probe. Btw, I got an error b/c replicas: {{ ((num_users|int) / 3) | int }} doesn't work.
fatal: [bastion.6b94.example.opentlc.com]: FAILED! => {"reason": "We were unable to read either as JSON nor YAML, these are the errors we got from each:\nJSON: Expecting value: line 1 column 1 (char 0)\n\nSyntax Error while loading YAML.\n  found unacceptable key (unhashable type: 'AnsibleMapping')\n\nThe error appears to be in '/Users/danieloh/RedHat-Middleware-Workshops/agnosticd/ansible/roles/ocp4-workload-ccnrd/tasks/install-mta.yaml': line 19, column 20, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n      spec:\n        replicas: {{ ((num_users|int) / 3) | int }}\n                   ^ here\nWe could be wrong, but this one looks like it might be an issue with\nmissing quotes. Always quote template expression brackets when they\nstart a value. For instance:\n\n    with_items:\n      - {{ foo }}\n\nShould be written as:\n\n    with_items:\n      - \"{{ foo }}\"\n"}

Reply:
Ok. I fixed this.